### ISSUE TITLE
BUG: Clipto v7.2.3 window handling problems 

### ISSUE THREAD

**Issue by klorinczi** [clipto-pro/Desktop#120]
There are window handling problems in Clipto v7.2.3.

The restore window button is not working, nothing happens clicking on it. 
The window dragging is not possible by dragging the top window bar.

Only minimize and close buttons are working.

Waiting for the fix.

Thanks!

**Comment by klorinczi** [comment]
Closings was just an accident.
Bug is alive.

**Comment by atrashler** [comment]
Could you please attach a screenshot of the window (or video of dragging if possible) and your platform?

**Comment by klorinczi** [comment]
A screenshot would not show anything special.

It is a maximized Clipto window, which is not possible to be restored, because clicking the restore icon has not effect.
Also dragging the maximized window is not possible, because it does not do anything on drag.

The only window functions working are: minimize, close.

Using Windows 7 Pro 64bit.

**Comment by atrashler** [comment]
> A screenshot would not show anything special.
> 
> It is a maximized Clipto window, which is not possible to be restored, because clicking the restore icon has not effect. Also dragging the maximized window is not possible, because it does not do anything on drag.
> 
> The only window functions working are: minimize, close.
> 
> Using Windows 7 Pro 64bit.

Got it. Could you please check:
1) if it is possible to maximise/minimise the window via double click on the toolbar
2) if it is possible to drag the toolbar when the app is minimised

**Comment by klorinczi** [comment]
> if it is possible to maximise/minimise the window via double click on the toolbar

No. Not even displayed any context menu, when the right click  mouse button is pressed. 


> if it is possible to drag the toolbar when the app is minimised

The Clipto window is displayed on the taskbar and it is possible to rearrange between groups with the left mouse button. 
Right mouse button press does not display context menu. 
Preview works, no right mouse button effect.

**Comment by atrashler** [comment]
> No. Not even displayed any context menu, when the right click mouse button is pressed.

Not right mouse click:) Double click. Double click now should maximise/minimise the window

**Comment by klorinczi** [comment]
I also meant, the double click had no effect either.

I will test the v7.2.6 for Windows, when it will be released.
